src/Mocap/MocapPoseDebugOverlay.cpp

```
@@ -0,0 +1,304 @@
+    return h > 1e-4f ? h : 1.65f;
+}
+
+void fkPoseIkJoints(
+    const std::array<std::array<float, 4>, PoseIK::kCanonicalRoles>& quats,
+    uint32_t resolvedMask,
+    const std::array<std::array<float, 3>, PoseIK::kLandmarkCount>& canonLmPts,
+    std::array<Vec3, PoseIK::kCanonicalRoles>& out)
+{
+    out.fill({0.f, 0.f, 0.f});
+    const Vec3 hip = mid(canonLmPts[23], canonLmPts[24]);
+    out[static_cast<size_t>(PoseIK::Hip)] = hip;
```

Cppcheck (2.21.0)

[error] Address of local auto-variable assigned to a function parameter.

(autoVariables)